sound/soc/codecs/max98373.c

```diff
@@ -24,7 +24,7 @@ static int max98373_dac_event(struct snd_soc_dapm_widget *w,
 	struct max98373_priv *max98373 = snd_soc_component_get_drvdata(component);
 
 	switch (event) {
-	case SND_SOC_DAPM_POST_PMU:
+	case SND_SOC_DAPM_PRE_PMU:
```

Comment:
PRE_PMU and POST_PMD looks symmetric to me, @ryans-lee any comment?

Reply:
@keyonjie `PRE_PMU` means the event will be trigger **before** the power bit (MAX98373_R202B_PCM_RX_EN) is set and `POST_PMU` means the event will be trigger **after** the power bit. PMU and PMD don't have to be symmetric.

Reply:
@bardliao do you know if we should set R20FF first or enable R202B first? looks we should enable the global (20FF) first and then enable the PCM_RX_EN? I don't know the codec well, but in case you power on A first and then B, for powering off, we should do with power off B first and then A? That is why symmetric I proposed.

Reply:
> @bardliao do you know if we should set R20FF first or enable R202B first? looks we should enable the global (20FF) first and then enable the PCM_RX_EN? I don't know the codec well, but in case you power on A first and then B, for powering off, we should do with power off B first and then A? That is why symmetric I proposed.

No, I don't know that. Maybe @ryans-lee can help.